app/controllers/public.js

```diff
@@ -20,6 +20,19 @@ export default class PublicController extends Controller {
     return this.session.currentRouteName && this.session.currentRouteName !== 'public.cfs.new-session' && this.session.currentRouteName !== 'public.cfs.new-speaker' && this.session.currentRouteName !== 'public.cfs.edit-speaker' && this.session.currentRouteName !== 'public.cfs.edit-session';
   }
 
+  @computed('locationName', 'online')
+  get headerLocation() {
+    if (this.model.locationName && this.model.online) {
+      return `In-Person Event and Online Event ${this.model.locationName}`;
+    } else if (this.model.online) {
+      return 'Online Event';
+    } else if (this.model.locationName) {
+      return this.model.locationName;
+    } else {
+      return 'Location to be announced';
+    }
+  }
+  
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6259866)